CMakeLists.txt

```diff
@@ -26,7 +26,14 @@ endif()
 
 include(CMakeDependentOption)
 
+set(DLAF_WITH_OPENMP_DESCRIPTION "Enable OpenMP support (TODO: Explanation for why?)")
```

Comment:
Do you think it's necessary to explain here what this is for? If yes, I think the explanation would be similar to the warning message on line 53, which is printed if `DLAF_WITH_OPENMP` is disabled _and_ `DLAF_WITH_MKL` is disabled. However, if we print the warning anyway I'm not sure if there's a need to have a long explanation here. What do you think?

Reply:
I removed the TODO and marking this as resolved. However, feel free to reopen if you feel that something needs more explanation.